### ISSUE TITLE
Can't compile project for web using with Flutter 3.7.0

### ISSUE THREAD

**Issue by BirjuVachhani** [BringingFire/rich_clipboard#15]
Can't compile my project for web using `flutter build web` with Flutter 3.7.0

```
../../../../.pub-cache/hosted/pub.dev/rich_clipboard_web-1.0.1/lib/rich_clipboard_web.dart:114:12: Error: `@staticInterop` classes should not contain any generative constructors.
Use factory constructors instead.
  external _ClipboardItem(dynamic args);
           ^
```

**Comment by BirjuVachhani** [comment]
I don't know why but adding this `factory` keyword fixes it!

![image](https://user-images.githubusercontent.com/20423471/214789843-6f3f500e-9c67-4564-9c34-25898a948abd.png)

**Comment by jmatth** [comment]
Underlying issue was dart-lang/sdk#48730, which shipped in Dart 2.19.
